src/main.js

```diff
@@ -16,102 +16,100 @@
 *    You can download the source here: https://github.com/spark/spark-server
 */
 
-var fs = require('fs');
-var http = require('http');
-var express = require('express');
-
-
-var settings = require('./settings.js');
-var utilities = require("./lib/utilities.js");
-var logger = require('./lib/logger.js');
+import bodyParser from 'body-parser';
+import express from 'express';
+import http from 'http';
+import morgan from 'morgan';
+import OAuthServer from 'node-oauth2-server';
+import { DeviceServer } from 'spark-protocol';
+import settings from './settings';
+
+import utilities from './lib/utilities';
+import logger from './lib/logger';
+import OAuth2ServerModel from './lib/OAuth2ServerModel';
+import AccessTokenViews from './lib/AccessTokenViews';
+import UserCreator from './lib/UserCreator.js';
+
+import api from './views/api_v1.js';
+import eventsV1 from './views/EventViews001.js';
+import Webhook from './views/Webhooks';
+
+const  NODE_PORT = process.env.NODE_PORT || 8080;
+
+// TODO wny do we need this? (Anton Puko)
+global._socket_counter = 1;
```

Comment:
I want to get rid of all the `global` calls.  It is really shitty practice.

I think what this is for is to keep track of each socket that gets opened in order to build an id or something